Code.js

```diff
@@ -255,6 +262,71 @@ function onMessage(event) {
     return createCardResponse(widgets);
   }
   
+    // If a user wants a new doc with their team's template
+  else if (strLowerCase.includes("new doc with template")) { 
+    var widgets = [{
+    "textParagraph": {
+      "text": "I have made you a new file with your team's template."
+    },
+    "buttons": [{
+      "textButton": {
+        "text": "Open new file",
+        "onClick": {
+          "openLink": {
+            "url": "https://docs.google.com/document/d/" + templateDocId + "/copy"
+          }
+        }
+      }
+    }]
+    }];
+
+    return createCardResponse(widgets);
+  }  
+  
+  // Give your team edit access to a doc or folder
+  else if (strLowerCase.includes("give team edit access to doc:") || strLowerCase.includes("give team edit access to folder:")) {
+    var Name = str.match(/"(.*?)"/g)[0];
+    Name = Name.substring(1, Name.length-1);
+    
```

Comment:
I like the regex, but I'm just wondering if you'll hit issues if there's no match? What happens if you write: @Chatbot give team edit access to doc:

Reply:
nit but should Name be all lowercase

Reply:
@miasya right now if the user doesn't use double quotes they'll get a message saying that the bot doesn't recognize the command. I think it's a good idea though to maybe have a separate error message if there aren't any quotes reminding the user to use quotes. As for the Name, I used the uppercase string because a user may want to name a new document/folder with capital letters in it.

Reply:
@SydneyCaulfeild Ok having the not recognized makes sense! Good idea for adding the other message, though I don't see it as a high priority thing.
So for Name, the uppercase is a reminder that it's mixed case?

Reply:
@miasya oh I didn't realize you were talking about the var name, I thought you were reminding me that the "str" variable is mixed case. I'll update the variable name!